### PR TITLE
Add Mountain America Credit Union

### PR DESCRIPTION
**Pull request by parkerbxyz**

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -624,6 +624,14 @@ websites:
       img: monzobank.png
       tfa: No
 
+    - name: Mountain America Credit Union
+      url: https://www.macu.com
+      img: macu.png
+      tfa: Yes
+      sms: Yes
+      email: Yes
+      doc: https://www.macu.com/home/services/online-banking/new-online-branch
```
Comment (jtagcat) [inline _data/banking.yml]:
```suggestion

```

**Review by jtagcat — COMMENTED** [review]
The doc doesn't actually give any information other than proof, that it exists.

File: _data/banking.yml [hunk 2]
```diff
@@ -630,7 +630,6 @@ websites:
       tfa: Yes
       sms: Yes
       email: Yes
-      doc: https://www.macu.com/home/services/online-banking/new-online-branch
```
Comment (kmpoppe) [inline _data/banking.yml]: The documentation you provided says "Receive MFA PINs through text, email or automated phone call.", so could you please add this option?

Comment (kmpoppe) [reply _data/banking.yml]:
```suggestion
      phone: yes
```

**Review by kmpoppe — CHANGES_REQUESTED** [review]

**Review by kmpoppe — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**